src/app/helpers/controller/cms-mixin.js

```diff
@@ -125,12 +125,17 @@ export function useTextFromSlug(slug) {
 
         fetch(url)
             .catch((err) => setText(err))
```

Comment:
should `catch` should come after the `.then`?  if it did then it'd also catch any JS errors that might occur inside the `then` block, like maybe if the html was invalid and parser failed?